tests/test_exclude.py

```diff
@@ -132,3 +132,47 @@ def test_config_exclude_with_subdirectory(
     build_file_tree({src / "copier.yml": "", src / "template" / "copier.yml": ""})
     run_copy(str(src), dst, quiet=True)
     assert not (dst / "template" / "copier.yml").exists()
+
+
+def test_config_exclude_without_templates_suffix(
+    tmp_path_factory: pytest.TempPathFactory,
+) -> None:
+    src, dst = map(tmp_path_factory.mktemp, ("src", "dst"))
+    build_file_tree(
+        {
+            (src / "copier.yml"): (
+                """\
+                _subdirectory: .
+                _templates_suffix: ""
+                _envops:
+                    block_start_string: "[?"
+                    block_end_string: "?]"
+                """
+            ),
+        }
+    )
+    run_copy(str(src), dst, quiet=True)
+    assert not (dst / "copier.yml").exists()
+
+    src, dst = map(tmp_path_factory.mktemp, ("src", "dst"))
```

Comment:
Shouldn't this be a separate test?

Reply:
Could be. I thought the `copier.yml` case is just a special instance of the underlying problem – and motivated by the bug report. I  replicated the test pattern from the test above. Would you prefer it moved into separate test?

Reply:
Yeah it looks like bundling 2 tests in 1. You just have to define a separate function before this line and we'll have more unitary tests. If one day one of them fails, it'll be easier to debug.

Reply:
Yeah, makes no sense to bundle 4 tests in one.